docs/marionette.features.md

```diff
@@ -6,6 +6,14 @@ Marionette Features are opt-in functionality. That you can enable by setting `Ma
 + make it possible to add breaking changes in a minor release
 + give community members a chance to provide feedback for new functionality
 
+## `childViewEventPrefix`
+
+This flag is set to `true` by default.
+
+This flag indicates whether [`childViewEventPrefix`](./events.md#a-child-views-event-prefix)
+for all views will return the default value of `'childview'` or if it will return `false`
+disabling [automatic event bubbling](./events.md#event-bubbling).
```

Comment:
I'm not sure how else we'd use these flags... right now they're all essentially
```js
if (flag) {
  doFlagThing();
} else  {
  doOtherThing();
}
```
So if you change the flag at some point after the start of your app, you're changing the behavior.